tests/utils.js

```diff
@@ -507,6 +509,26 @@ const createUsers = async (users, meta = false) => {
   }
 };
 
+const createUsersNative = async (users, meta = false) => {
```

Comment:
I've just merged a PR that removes the need for such methods in this utils file. You can just use the existent methods and they should work fine.

Reply:
Done